tests/dummy/app/docs/typescript/template.hbs

```diff
@@ -0,0 +1,157 @@
+<h3>Using ember-concurrency with TypeScript</h3>
+
+<p>
+  As of version 1.2, ember-concurrency comes bundled with its own
+  <a href="https://github.com/machty/ember-concurrency/blob/master/addon/index.d.ts">
+  type definitions</a>. TypeScript should automatically pick them
+  up to provide type checking and completion.
+</p>
+
+<p>
+  Note that old versions of <a href="https://github.com/machty/ember-concurrency-decorators/">
+  ember-concurrency-decorators</a> came with an incomplete type
+  definition for ember-concurrency. If you are using that addon,
+  be sure to upgrade to the latest version to avoid conflicts.
```

Comment:
I assumed this as well. I'm not sure what the version number would be, so I just said "latest". There is a 1.1 "alpha" that is pretty unrelated, so we may want to skip 1.1 to avoid confusion. Jumping to 1.2 would also match up with the e-c version number, a happy accident.

Reply:
(still relevant)